packer/install-dependencies.sh

```diff
@@ -64,8 +64,8 @@ installGPUDrivers
 echo "  - nvidia-docker2 nvidia-container-runtime" >> ${VHD_LOGS_FILEPATH}
 
 VNET_CNI_VERSIONS="
+1.0.29
```

Comment:
It looks like we need to update `AZURE_NPM_VERSIONS` in this same file–it still points to 1.0.18. And I don't see us fetching `azure-vnet-telemetry` anywhere either, so that should probably be added here as well.

Reply:
updated